emp/emp_iam_cftemplate.yml

```diff
@@ -52,6 +52,7 @@ Resources:
           - ec2:DescribeAvailabilityZones
           - ec2:DescribeImages
           - ec2:DescribeInstances
+          - ec2:DescribeInstanceAttribute
```

Comment:
Update the aws-policy.json as well. Can you validate that both of these files are in sync?

Reply:
Sure will also check that. Checked for cloud formation template.